Essentials/src/com/earth2me/essentials/commands/Commandtpa.java

```diff
@@ -35,8 +38,12 @@ public void run(Server server, User user, String commandLabel, String[] args) th
             && player.isTpRequestHere() == false) { // Make sure the last teleport request was actually tpa and not tpahere
             throw new Exception(tl("requestSentAlready", player.getDisplayName()));
         }
-
+        TPARequestEvent tpaEvent = new TPARequestEvent(user.getSource(), player, false);
         if (!player.isIgnoredPlayer(user)) {
+        	ess.getServer().getPluginManager().callEvent(tpaEvent);
+            if(tpaEvent.isCancelled()) {
```

Comment:
format

Reply:
Also, why are we constructing the event without knowing if we're going to use it for sure or not?

Reply:
Apologies, aforementioned issues will be fixed in the next commit.